packages/database/src/database/donations.ts

```diff
@@ -133,7 +133,7 @@ export async function getDonationById(
   return data
 }
 
-export async function newDonation(data: Prisma.DonationCreateInput) {
+export async function newDonation(data: Prisma.DonationUncheckedCreateInput) {
```

Comment:
Not sure about this change... Although it's unlikely, it's possible to put an incorrect foreign ID, which this protects against. IMO it's better to use the `connect` syntax as described here: https://www.prisma.io/docs/orm/prisma-client/type-safety

I'll put an example with the donation.

Reply:
Temp fix to make it run, typing was giving me headaches as the organization type used had wallets and all, so it wouldn't let me use it as part of the donation record. We can chat in slack about how to make it work